src/components/Grid/Grid.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Grid({ children }) {
+  return <AppGrid className="Grid"> {children} </AppGrid>
```

Comment:
Are the spaces here needed? If not I would delete those. 
You can also delete the className since your using styled components. :D